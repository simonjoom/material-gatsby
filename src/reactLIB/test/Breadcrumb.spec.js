import React from 'react';
import { shallow } from 'enzyme';
import Breadcrumb from '../src/Breadcrumb';

describe('<Breadcrumb />', () => {
  test('renders', () => {
    const wrapper = (
      <Breadcrumb cols={6} className="blue">
        <a href="#">Home</a>
        <a href="#">Login</a>
      </Breadcrumb>
    );
    expect(shallow(wrapper)).toMatchSnapshot();
  });
});
