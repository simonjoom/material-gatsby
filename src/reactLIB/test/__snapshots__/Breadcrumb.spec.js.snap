// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Breadcrumb /> renders 1`] = `
<nav
  className="row blue"
>
  <div
    className="nav-wrapper"
  >
    <Col
      s={6}
    >
      <a
        className="breadcrumb"
        href="#"
        key=".0"
      >
        Home
      </a>
      <a
        className="breadcrumb"
        href="#"
        key=".1"
      >
        Login
      </a>
    </Col>
  </div>
</nav>
`;
